lang/cs/Org.Apache.REEF.Driver/Task/TaskStates.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.Driver.Task
+{
+    /// <summary>
+    /// Task states in task transitions
+    /// </summary>
+    public enum TaskStates
+    {
+        TaskNew,
+        TaskSubmitting,
+        TaskRunning,
+        TaskCompeleted,
+        TaskWaitingForClose,
+        TaskClosedByDriver,
+        TaskFailedByEvaluatorFailure,
+        TaskFailedByGroupCommunication,
+        TaskFailedAppError,
+        TaskFailedSystemError
+    }
+
+    /// <summary>
+    /// Task events that triggers task state transition
+    /// </summary>
+    public enum TaskEvents
+    {
+        SubmittingTask,
+        RunningTask,
+        CompleteTask,
+        WaitingTaskToClose,
+        CloseTask,
+        FailTaskAppError,
+        FailTaskSystemError,
+        FaileTaskEvaluatorError,
+        FailTaskCommuError
```

Comment:
Please don't use abbreviations here. IntelliSense will ease the burden of typing the full words :)